config/externalnamenottested.go

```diff
@@ -310,24 +334,39 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 
 	// cloudfunctions2
 	//
+	// Imported by using the following projects/{{project}}/locations/{{location}}/functions/{{name}}
+	"google_cloudfunctions2_function": config.TemplatedStringAsIdentifier("name", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.location }}/functions/{{ .external_name }}"),
```

Comment:
We may consider marking the `location` field as required.

Reply:
Done